docs/advanced_reference/commands/autonomy_deploy.md

```diff
@@ -0,0 +1,196 @@
+
+## Usage
+
+```bash
+autonomy deploy [OPTIONS] COMMAND [ARGS]
+```
```

Comment:
No need for a usage section on a parent command unless there are some top level flags